config/task_test.go

```diff
@@ -374,10 +387,37 @@ func TestTaskConfig_Validate(t *testing.T) {
 			false,
 		},
 		{
-			"missing service",
+			"no service with no condition",
 			&TaskConfig{Name: String("task"), Source: String("source")},
 			false,
 		},
+		{
+			"no service with services condition",
+			&TaskConfig{
+				Name:      String("task"),
+				Source:    String("source"),
+				Condition: &ServicesConditionConfig{},
+			},
+			false,
+		},
+		{
+			"missing service with catalog-service condition and no regexp",
+			&TaskConfig{
+				Name:      String("task"),
+				Source:    String("source"),
+				Condition: &CatalogServicesConditionConfig{},
+			},
+			false,
+		},
+		{
+			"invalid catalog-service condition (bad regexp)",
+			&TaskConfig{
+				Name:      String("task"),
+				Source:    String("source"),
+				Condition: &CatalogServicesConditionConfig{Regexp: String("*")},
+			},
+			false,
```

Comment:
this test case is unrelated to the feature. i noticed it missing when looking at coverage so snuck it in 🙏

Reply:
out of curiosity, what error is surfaced when `*` is set?

Reply:
Yea, it surprised me that `*` errors. `regexp.Compile("*")` returns the error "error parsing regexp: missing argument to repetition operator: `*`"

Reply:
oh cool ok that seems informative enough that it should be `.*` for ppl somewhat familiar with regex. Thanks!

Reply:
Yea, agree the message is pretty helpful. Thanks for checking on this! @ eikenb also suggested, which I thought would be really helpful for users, that I add a link to a regex intro or something in our documentation. There've been a couple other regex surprises for me personally